lib/webtask.js

```diff
@@ -708,6 +736,92 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function check_against_secret(jwt, callback) {
+            try {
+                auth_token = req.headers['authorization']
+                .match(/\s.+$/)[0]
+                .replace(/\s/, '');
+            } catch(e) {
+                msg = { code: 401, message: 'Unauthorized to run webtask. Could not parse authorization header.', req_id: req.req_id };
+                return callback(msg);
+            }
+
+            try {
+                decoded_auth_token = jsonwebtoken.verify(auth_token, jwt.payload.auth.secret);
```

Comment:
This appears to be a free-floating global that is only defined in another scope below.